proto/raze/crates.bzl

```diff
@@ -3,26 +3,26 @@ cargo-raze crate workspace functions
 
 DO NOT EDIT! Replaced on runs of cargo-raze
```

Comment:
This is a generated file, I think there is an open issue that cargo raze is not respecting buildifier format...

Since it seems only to be formatting fix, maybe we can revert that file for now?

Reply:
Reverted.